docs/CIRCUIT_BREAKER_GUIDE.md

```
@@ -0,0 +1,162 @@
+## Monitoring
+
+### Health Check Endpoint
+
+The `/health` endpoint now includes circuit breaker status:
+
+```bash
+curl http://localhost:4000/health
+```
+
+Response includes:
+```json
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)